schema.graphql

```diff
@@ -77,4 +81,33 @@ type Settlement @entity {
   firstTradeTimestamp: BigInt!
   "Collection of trades"
   trades: [Trade!] @derivedFrom(field: "settlement")
-}
\ No newline at end of file
+}
+
+# stores for USD calculations
+type Bundle @entity {
```

Comment:
same with the naming. Not sure why they call this Bundle. I personally think is clearer `EthUsdPriceSingleton`

Reply:
Sure!